integrations/authentication/OAuth2Manager.js

```diff
@@ -58,7 +60,7 @@ export default class {
       redirect_uri: getCallbackUrl(),
       response_type: 'code',
     });
-    const authorizationCodeRequestUrl = `${this.authorizeUrl}?${requestURLParams}`;
+    const authorizationCodeRequestUrl = `${this.authorizeUrl}?${requestURLParams}&scope=${this.scope}`;
     const authorizationResponseQuery = await openUrlAndWaitForCallback(
       authorizationCodeRequestUrl,
       getCallbackUrl()
```

Comment:
I had left out the scope param out of `objectToURLParams` because the method converts colons from the scope string to their URL encoding equivalents, which are not accepted by Automatic's API and break the authorization functionality. So, I just attached the scope param outside the `objectToURLParams` function in the latest commit; Uber should work now 😊

Reply:
Okay, that makes sense. Can you add a comment explaining this? 

Also, let's just revert the change to objectToURLParams.js now that it isn't needed

Reply:
Ok, updated it in latest commit